frontend/pages/hosts/details/HostDetailsPage/modals/OSPolicyModal/OSPolicyModal.tsx

```diff
@@ -41,7 +41,7 @@ const RenderOSPolicyModal = ({
         .catch(() => setCopyMessage("Copy failed"));
 
       // Clear message after 1 second
```

Comment:
same here with comment needing updating

Reply:
oofph, I accidentally left the 6 seconds I used to make it easier to screenshot the 1 second message! Fixed!